net/uplog/Portfile

```diff
@@ -1,28 +1,32 @@
-PortSystem      1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name				uplog
-version				0.3
+PortSystem              1.0
+PortGroup               github 1.0
 
-categories			net
-maintainers			nomaintainer
-platforms			darwin
+github.setup            carragom uplog 0.3
+github.tarball_from     archive
+revision                1
+categories              net
+license                 GPL-2.0
+maintainers             nomaintainer
 
-description			A UDP based ping program that can give an ASCII-graphical \
-				log of the packet losses.
+description             A UDP based ping program that can give an ASCII-graphical log of the packet losses
+long_description        ${description}.
 
-long_description		${description}
+checksums               rmd160  246a4668beae74505c50228a57e54ab7c4c57f13 \
+                        sha256  fdf8b64835023525cbaefac750493dc7ee265bbcc07f4081105c5f484dc8682a \
+                        size    50720
 
-homepage			http://www.nerdlabs.org/projects/uplog.php
-master_sites		        http://www.nerdlabs.org/downloads/uplog/
-checksums			md5 38af7eab6c656befbda24683124ac8dc
+patch.pre_args-replace  -p0 -p1
+patchfiles              implicit.patch
+patchfiles-append       fix-configure-script.patch
 
-post-patch {
-	reinplace "s|#include <malloc.h>||g" \
-		${worksrcpath}/uplog.c
-}
+depends_build-append    port:autoconf \
+                        port:automake \
+                        port:libtool
```

Comment:
why are these needed? It doesn't look like you're regenerating any files but just use the `configure` script that is present already?